src/posit/connect/resources.py

```diff
@@ -125,9 +123,16 @@ def _data(self) -> List[T]:
         if self._cache:
```

Comment:
I believe empty lists are also false here. 

We should probably compare against `self._cache is not None`